library/pom.xml

```diff
@@ -306,12 +306,24 @@
       <artifactId>janino</artifactId>
       <version>2.7.8</version>
       <scope>test</scope>
-    </dependency>    
-     <dependency>
+    </dependency>
+    <dependency>
```

Comment:
Looks like junit dependency is already available as a transitive dependency otherwise the tests wouldn't have compiled so far. Why this new addition?